core/src/test/java/com/google/googlejavaformat/java/FormatterTest.java

```diff
@@ -249,6 +249,15 @@ public void importsNotReorderedByDefault() throws FormatterException {
   }
 
   @Test
+  public void importsOptimizedIfRequested() throws FormatterException {
+    String input =
+      "package com.google.example;\n" + UNORDERED_IMPORTS + "\npublic class ExampleTest {}\n";
```

Comment:
@sormuras Does not Google Java Style dictate that indents in multiline statements must be +4, or am I missing something here?